sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -344,6 +348,30 @@ export class ApplicationOfferingChangeRequestService {
     return applicationOfferingChangeRequest;
   }
 
+  /**
+   * Get the Application Offering Change Request Status by id.
+   * @param applicationOfferingChangeRequestId the application offering change request id.
+   * @param options method options:
+   * - `studentId`: student id for student authorization.
+   * @returns application offering change request status.
+   */
+  async getApplicationOfferingChangeRequestStatusById(
```

Comment:
IMO, it would be better not to use  `byid for the function name, unless its a api service layer

Reply:
I am sorry, I didn't get this. It is a service layer method.